examples/metatensor-plumed/lj-crystal.py

```diff
@@ -0,0 +1,543 @@
+featurizer = chemiscope.metatomic_featurizer(model)
+chemiscope.explore([minimal, other, atoms], featurize=featurizer, settings=settings)
+
+# %%
+#
+# Using the model to run metadynamics with PLUMED
+# -----------------------------------------------
+#
+# With our model saved, we can now write the PLUMED input file. This file
+# instructs PLUMED on what to do during the simulation.
+# The input file consists of the following sections:
+# - `UNITS` : Specifies the energy and length units
+# - `METATOMIC` : Defines a collective variable which is essentially
+#                 an exported metatomic model
+# - `SELECT_COMPONENTS` : Splits the model output :math:`Q_4`
+#                         and :math:`Q_6` parameters to scalars
+# - `METAD` : sets up the metadynamics algorithm. It will add repulsive Gaussian
+#             potentials in the (`cv1`, `cv2`) space at regular intervals (`PACE`),
+#             discouraging the simulation from re-visiting conformations and pushing it
+#             over energy barriers
+# - `PRINT` : This tells PLUMED to write the values of our CVs and the
+#             metadynamics bias energy to a file named `COLVAR` for later analysis.
+
+if os.path.exists("HILLS"):
+    os.unlink("HILLS")
+
+setup = [
+    f"UNITS LENGTH=A ENERGY={ase.units.mol / ase.units.kJ}",
+    # define a collective variables using metatensor
+    """
+    cv: METATOMIC
+        MODEL=custom-cv.pt
+        EXTENSIONS_DIRECTORY=./extensions/
+        SPECIES1=1-38
+        SPECIES_TO_TYPES=18
+    """,
+    # extract the different components from the METATOMIC output into scalars
+    # (METAD only accepts scalars, and METATOMIC output is a vector here)
+    "cv1: SELECT_COMPONENTS ARG=cv COMPONENTS=1",
+    "cv2: SELECT_COMPONENTS ARG=cv COMPONENTS=2",
+    # run metadynamics with this collective variable
+    """
+    mtd: METAD
+        ARG=cv1,cv2
+        HEIGHT=0.05
+        PACE=50
+        SIGMA=1,2.5
+        GRID_MIN=-20,-40
+        GRID_MAX=20,40
+        GRID_BIN=500,500
+        BIASFACTOR=5
+        FILE=HILLS
+    """,
+    # prints out trajectory
+    """
+    PRINT ARG=cv.*,mtd.* STRIDE=10 FILE=COLVAR
+    """,
+    """
+    FLUSH STRIDE=1
+    """,
+]
+
+# %%
+# Running dynamics I - `ase`
+# ---------------------------
+#
+# The easiest way to generate a trajectory is to leverage `ase`. In subsequent
+# sections we will use LAMMPS, as a more production worthy and correct molecular
+# dynamics engine.
+#
+
+# Create the Plumed calculator, which wraps the LJ potential
+atoms.calc = ase.calculators.plumed.Plumed(
+    calc=lj_potential,
+    input=setup,
+    timestep=0.01,
+    atoms=atoms,
+    kT=0.1,
+)
+atoms.set_masses([1.0] * len(atoms))
+
+
+md = ase.md.langevin.Langevin(
+    atoms,
+    timestep=0.01,
+    temperature_K=0.1 / ase.units.kB,
+    friction=1.0,
+)
+
+trajectory = [atoms.copy()]
+for _ in range(100):
+    md.run(steps=10)
+    trajectory.append(atoms.copy())
+
```

Comment:
Do we want to do something with this trajectory? Otherwise accumulating the atoms around is not really needed

Reply:
It is used for the pretty picture and the interactive view in `chemiscope`.